libs/langchain/langchain/storage/in_memory.py

```diff
@@ -0,0 +1,79 @@
+"""In memory store that is not thread safe and has no eviction policy.
+
+This is a simple implementation of the AbstractStore using a dictionary that is useful
+primarily for unit testing purposes.
+"""
+from typing import Any, Dict, Iterator, List, Optional, Sequence, Tuple
+
+from langchain.schema import BaseStore
+
+
+class InMemoryStore(BaseStore[str, Any]):
+    """In-memory implementation of the AbstractStore using a dictionary.
+
```

Comment:
should we note Attribute `store` (or make it private)

Reply:
documented the attribute, should probably expose it since it's mostly for testing